app/src/main/java/com/valentin/secondhomework/model/service/dataService/DataServiceProvider.kt

```diff
@@ -0,0 +1,38 @@
+package com.valentin.secondhomework.model.service.dataService
+
+import androidx.lifecycle.MutableLiveData
+import com.valentin.secondhomework.model.data.RandomItemwq
+
+class DataServiceProvider : DataService {
+
+    private val randomItemList = MutableLiveData<List<RandomItem>>()
+
+
+    override fun getData(): MutableLiveData<List<RandomItem>> {
+        if (isRandomItemListEmpty())
+        //if list is empty, get data from the api
```

Comment:
// Comments should be used ONLY to highlight, notify, raise attention to other developers that something is tricky, requires special attention, or is just hard to understand (and cannot be simplified)
// You should NEVER commit code with todo's 
// You should never add logs to make your code more clear -> instead, you should clean the code